src/test/groovy/com/ofg/geb/AcceptanceSwaggerUISpec.groovy

```diff
@@ -0,0 +1,54 @@
+package com.ofg.geb
+
+import com.ofg.geb.pages.SwaggerUIHomePage
+import com.ofg.twitter.Application
+import geb.spock.GebSpec
+import org.springframework.boot.test.IntegrationTest
+import org.springframework.boot.test.SpringApplicationContextLoader
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.web.WebAppConfiguration
+
+@ContextConfiguration(loader = SpringApplicationContextLoader.class, classes = Application)
+@WebAppConfiguration
+@IntegrationTest("spring.profiles.active:dev,stubrunner.skip-local-repo:true")
+class AcceptanceSwaggerUISpec extends GebSpec {
+
+    def "SwaggerUI home page should be visible"() {
+        when:
+        to SwaggerUIHomePage
```

Comment:
I would propose more compact Spock specification formatting already used n other places: https://solidsoft.wordpress.com/2014/05/20/more-compact-automatic-spock-specification-formatting-in-idea-13/